### DISCUSSION TITLE
improve drive-mode ✋

### DISCUSSION THREAD

**Discussion by lsaavedr** [makeswift/makeswift#806] [Ideas]
Hi,

when we work with a site with a login, we need to pass them the credentials or some interactions first to access the page we need to edit, in this context is when we switch to ✋, to enter the credentials, but there is missing persistent cookies or another source of data in the frame, and this makes it impossible to access the page

regards!

**Comment by migueloller** [comment]
@lsaavedr, are the cookies perhaps not being stored due to browser cookie settings? Note that since the site is in an `<iframe>` in `app.makeswift.com` and the origin for your domain will be different.

Maybe you can share a bit more of your setup and the specific issues you're running into?
